jobs/config.json

```diff
@@ -8180,6 +8180,7 @@
       "--kubernetes-anywhere-kubelet-ci-version=latest",
       "--kubernetes-anywhere-kubernetes-version=ci/latest",
       "--provider=kubernetes-anywhere",
+      "--tag=latest",
```

Comment:
this `--tag` flag is not going to work for prow jobs..

Reply:
Will remove